nucleus/modelci/unit_test_evaluation.py

```diff
@@ -0,0 +1,40 @@
+"""Data types for Unit Test Evaluation results."""
+from enum import Enum
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class UnitTestEvaluationStatus(str, Enum):
+    """Job status of unit test evaluation."""
+
+    PENDING = ("pending",)
```

Comment:
Curious why we need to write this as a single-item tuple 🤔 

Seems neat that we're treating it as both `str` and `Enum` though.

Reply:
Not sure, this is copied from code that @sasha-scale previously wrote.

Reply:
I just changed them to strings, which I think should be ok.